libraries/SITL/SIM_Rover.h

```diff
@@ -44,7 +45,10 @@ class SimRover : public Aircraft {
     float turning_circle;
     float skid_turn_rate;
     bool skid_steering;
+    bool use_iceengine;
 
+    ICEngine icengine{2, 14, 12, 13, 100};
```

Comment:
Yowsers!  Magic numbers much?!

Reply:
I blame Jaime.

Reply:
Hey I’m just copying what the code gurus all ready did for plane if I knew what the numbers meant I would have changed them. Check the plane SITL code